scripts/qiita

```diff
@@ -59,9 +62,24 @@ def pet():
     pass
 
 
-@pet.group()
-def webserver():
-    pass
+@pet.group(invoke_without_command=True)
```

Comment:
Just to make sure. If I then run `qiita pet webserver start` will build the docs and start the server? and then `qiita pet webserver start --no-build-docs` will only start the server?